build.sbt

```diff
@@ -33,7 +33,7 @@ onLoad in Global := { s =>
 }
 
 // Uncomment for local gmp testing
-// resolvers in ThisBuild += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/Projects/maven-repo/releases"
+//resolvers in ThisBuild += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"
```

Comment:
This will not work in my Linux machine :(

Reply:
well, it is commented and it is mostly for me. Why wouldn't it work in Linux anyway?